dts/bindings/interrupt-controller/cypress,psoc6-intmux-ch.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2019, Linaro Limited
+# Copyright (c) 2020, ATL Electronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: Cypress Interrupt Multiplex Channel
+
```

Comment:
Can we add more description either here or in the `cypress,psoc6-intmux` binding on how this works and is used by a user.  ie the bits of setting the `interrupt-parent` for a device that is targeted to the M0+ to once of the `cypress,psoc6-intmux-ch`, etc.

Reply:
Add documentation at cypress,psoc6-int-mux.yaml